indico/legacy/webinterface/tpls/latex/inc/document.tpl

```diff
@@ -15,7 +15,7 @@
 \usepackage[a4paper, top=1em, bottom=10em]{geometry}
 </%block>
 
-\usepackage{hyperref}
+\usepackage[ocgcolorlinks]{hyperref}
```

Comment:
What is the minium required texlive version currently? 
If a new enough version ( > 2017..) is allowed, then 
`\usepackage{hyperref}
\usepackage[ocgcolorlinks]{ocgx2}[2017/03/30]`
would be a n even better option. (see [
_
TeX - LaTeX Stack Exchange](https://tex.stackexchange.com/questions/4425/)

Reply:
Since we did the stable release of 2.0 this year and texlive always installs the latest version (you can't even easily install an older one), I'd say we can safely assume people have at least 2017.